spec-tests/src/main/java/org/candlepin/spec/bootstrap/data/builder/Products.java

```diff
@@ -20,20 +20,24 @@
 import org.candlepin.dto.api.v1.ProductDTO;
 import org.candlepin.spec.bootstrap.data.util.StringUtil;
 
+import java.util.Random;
+
 /**
  * Class meant to provide fully randomized instances of product.
  *
  * Individual tests can then modify the instance according to their needs.
  */
 public final class Products {
 
+    private static final int MAX_ID = 10000;
+
     private Products() {
         throw new UnsupportedOperationException();
     }
 
     public static ProductDTO random() {
         return new ProductDTO()
-            .id(StringUtil.random("test_id"))
+            .id(String.valueOf(new Random().nextInt(MAX_ID)))
```

Comment:
This is one area where @Ceiu has also changed, so whichever PR gets merged first, the other one will have to adjust (and I think @Ceiu's change is more complete since it takes care of both engineering and market product id formats)

Reply:
Yeah, I agree! I think using the new Products.randomSKU() would make more sense.

Reply:
the pool_resource_spec PR is merged, so you can rebase and use that now

Reply:
Awesome. I updated to use the new method.